libraries/ms-common/src/can_ack.c

```diff
@@ -0,0 +1,116 @@
+// Uses an object pool to track the storage for ack requests, but the actual requests are handled
+// through an array of request pointers to minimize copying
+// ACK requests currently ordered as they were created
+#include "can_ack.h"
+#include "log.h"
```

Comment:
No. I don't really mind leaving `log.h` since it's helpful for debugging, but I'll remove it.